src/exampleTypes/TermsStatsTable.js

```diff
@@ -1,13 +1,80 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
 import React from 'react'
 import { observer } from 'mobx-react'
 import { exampleTypes } from 'contexture-client'
 import injectTreeNode from '../utils/injectTreeNode'
-import ExpandableTable from '../layout/ExpandableTable'
+import ExpandableTable, { Column } from '../layout/ExpandableTable'
 
 let TermsStatsTable = injectTreeNode(
-  observer(({ node, ...props }) => (
-    <ExpandableTable {...props} data={node.context.terms} />
-  )),
+  observer(
+    ({
+      node,
+      criteria,
+      tree,
+      children,
+      MoreControls = 'div',
+      Input = 'input',
+      ...props
+    }) => (
+      <div>
+        <div>
+          Filter:
+          <Input
+            type="text"
+            value={node.filter}
+            {...F.domLens.value(tree.lens(node.path, 'filter'))}
+          />
+        </div>
+        <ExpandableTable
+          {...{
+            ...props,
+            children: [
+              ...children,
+              <Column
+                label="Controls"
+                expand={{
+                  display: (value, record) => (
+                    <div>
+                      <button
```

Comment:
We should only do this if criteria is provided - if there is no criteria, this will break (and criteria should be optional)